### ISSUE TITLE
Statusbar cutting in iOS simulator after build

### ISSUE THREAD

**Issue by frisone** [nolimits4web/Framework7#1080]
This is a (multiple allowed):
- [x] bug
- [ ] enhancement
- [ ] feature-discussion (RFC)
- Framework7 Version: nobutakaoshiro:framework7-ios-material.
- Platform and Target: IOS.
### What you did

Resolve when statusbar show, the toolbar displayed cutting
### Expected Behavior

Statusbar don't cutting when statusbar show.
### Actual Behavior

When statusbar show, the toolbar displayed cutting.

P.S. Remember, an issue is not the place to ask questions. You can use [Stack Overflow](http://stackoverflow.com/questions/tagged/framework7)
for that or join the #framework7 channel on irc.freenode.net, where we will be more
than happy to help answer your questions.

Before you open an issue, please check if a similar issue already exists or has been closed before.

**Comment by valnub** [comment]
Hi, sorry for the late answer. I didn't really understand your problem. Could you provide a screenshot? Thanks.

**Comment by scriptPilot** [comment]
With [App Framework demo](http://app-framework.scriptpilot.de/) i have the same issue on iPhone SE.

![image 7](https://cloud.githubusercontent.com/assets/19615586/22228085/1315c37a-e1cf-11e6-9090-307f2fc9954a.png)

**Comment by scriptPilot** [comment]
PS: Only after pinned it to the homescreen

**Comment by valnub** [comment]
Hm, yes I can confirm the problem.

Maybe this helps? https://framework7.io/docs/status-bar.html

Did you try this?
```
<div class="statusbar-overlay"></div>
```
and

```
.statusbar-overlay {
    background: transparent;
}
```

Also, you might wanna try this: https://muut.com/framework7#!/framework7/bugs-and-issues:status-barlogin-screen-iss

`<meta name="apple-mobile-web-app-status-bar-style" content="black-translucent">`

**Comment by scriptPilot** [comment]
@valnub - Thank you, fixed it for F7, but not for F7 - I opened a new ticket nolimits4web/Framework7-Vue#71
